subgraph/tests/subscriptions.test.ts

```diff
@@ -39,24 +39,28 @@ import {
 // Tests structure (matchstick-as >=0.5.0)
 // https://thegraph.com/docs/en/developer/matchstick/#tests-structure-0-5-0
 
+const INITIAL_START = BigInt.fromU32(2000);
+const INITIAL_END = BigInt.fromU32(5000);
+const INITIAL_RATE = BigInt.fromU32(10)
+  .pow(18)
+  .times(BigInt.fromU32(2));
+
 describe('Describe entity assertions', () => {
   const user = '0x0000000000000000000000000000000000000001';
   const timestamp = 1; // default block timestamp
 
-  beforeAll(() => {
+  beforeEach(() => {
```

Comment:
I made the test cases independent from each other, because they were pretty annoying to debug otherwise. Couldn't isolate them (by commenting out other tests, shamefully) to fix one by one before.

Reply:
I ran into that as well. Thanks for doing this